dpnp/backend/extensions/fft/common.hpp

```diff
@@ -111,8 +111,12 @@ class DescriptorWrapper
         const typename valT::value_type dim = get_dim();
 
         valT fwd_strides(dim + 1);
+#if COMPILER_VERSION_FLAG
+        descr_.get_value(mkl_dft::config_param::FWD_STRIDES, &fwd_strides);
+#else
         descr_.get_value(mkl_dft::config_param::FWD_STRIDES,
                          fwd_strides.data());
+#endif
```

Comment:
Need to add/update comments after each `#endif`
```suggestion
#endif // COMPILER_VERSION_FLAG
```